packages/kitchen-sink/src/examples/sudoku.tsx

```diff
@@ -0,0 +1,504 @@
+ *          Generate Sudoku Board Component
+ *************************************************/
+export function generateSudokuBoard(): Board {
+  const board: Board = [];
+
+  for (let i = 0; i < 9; i++) {
+    board[i] = [];
+    for (let j = 0; j < 9; j++) {
+      board[i][j] = 0;
+    }
+  }
+
+  fillBoard(board, 0, 0);
+
+  return board;
+}
+
+/**************************************************
+ *             CheckSovled Component
+ *************************************************/
+ function checkSolved(board: Board, originalBoard: Board): boolean {
+  if (board.length !== 9 || originalBoard.length !== 9) {
+    return false;
+  }
+
+  for (let i = 0; i < 9; i++) {
+    for (let j = 0; j < 9; j++) {
+      if (board[i][j] !== originalBoard[i][j]) {
+        return false;
+      }
+    }
+  }
+  return true;
+}
+
+/**************************************************
+ *              NullifyValues Component
+ *************************************************/
+ const nullifyValues =  (board: Board) => {
+  const cellsToNullify = 40;
+  let cellsNullified = 0;
+  while (cellsNullified < cellsToNullify) {
+    const row = Math.floor(Math.random() * 9);
+    const col = Math.floor(Math.random() * 9);
+    if (board[row][col] !== 0) {
+      board[row][col] = 0;
+      cellsNullified++;
+    }
+  }
+};
+
+/**************************************************
+ *                Title Component
+ *************************************************/
+const Title = () => {
+  return <h1 className="title">Sudoku</h1>;
+};
+
+/**************************************************
+ *                Toast Component
+ *************************************************/
+const Toast = block(({ message }: { message: string }) => {
+  const [showToast, setShowToast] = useState(true);
+
+  useEffect(() => {
+    const timeout = setTimeout(() => {
+      setShowToast(false);
+    }, 3000); // 3000 milliseconds (3 seconds) to automatically close the toast
+
+    return () => {
+      clearTimeout(timeout);
+    };
+  }, []);
+
+  return (
+    <div className={`toast ${showToast ? 'show' : ''}`}>
+      <div className="toast-content success">{message}</div>
+    </div>
+  );
+});
+
+/**************************************************
+ *                Main Component
+ *************************************************/
+const App =() => {
+  const [initialBoard, setInitialBoard] = useState<Board>([]);
+  const boardOriginal = useRef<Board>(generateSudokuBoard());
+  const [board, setBoard] = useState<Board>([]);
+
+  useEffect(() => {
+    const newBoard = JSON.parse(JSON.stringify(boardOriginal.current)) as Board;
+    nullifyValues(newBoard);
+    setInitialBoard(newBoard);
+    setBoard(newBoard);
+  }, [boardOriginal]);
+
+ 
+  return (
+    <main
+      style={{
+        background:'linear-gradient(to bottom right, rgb(40, 40, 43),rgb(48, 25, 52))',
+        position : 'relative'
+      }}
+    >
+      <style>
```

Comment:
Your styles are kind of causing a problem. If you could review this @ANKITy102; 

https://github.com/aidenybai/million/assets/96334363/cf75d13e-f7c0-46a6-8884-87dca0069bc1

Reply:
@tobySolutions 
I have fixed the issue. You can verify. Thank you!

Reply:
> Your styles are kind of causing a problem. If you could review this @ANKITy102;
> 
>  Screen.Recording.2023-10-14.at.20.41.14.mov

@tobySolutions 
I have fixed the styling issue. Below is the video for the same

https://github.com/aidenybai/million/assets/119873198/3726d538-43ef-4cad-8824-0f83a3f540cc